HACKING.md

```diff
@@ -233,6 +233,13 @@ You need to disable SELinux with
 for this to work, as your local build tree does not otherwise have the expected
 SELinux type.
 
+Note that `./configure` defaults might not represent the running system's,
+cockpit-bridge might look in the wrong directory for libexecdir executables.
```

Comment:
cockpit-bridge shouldn't call any libexecdir, but I suppose it will not look into /usr/share/cockpit/ for finding the package-installed cockpit manifests. Check with `cockpit-bridge --packages`. It should run, though? Or is it failing to link, etc.?

This is true for cockpit-ws, though -- that will look for /usr/libexec/cockpit-session at least. So perhaps

> `./configure` defaults don't work with files installed from packages, so `cockpit-bridge` and `cockpit-ws` will look in the wrong directories for libexecdir executables or manifests.

Reply:
Ah, I went with:

```
[jelle@t14s][~/projects/cockpit]%strings /usr/bin/cockpit-bridge| grep libexec
libexecdir
/usr/lib/cockpit
```

So it at least records it, not sure if it does anything with it. Maybe we should just generalise the text?

"Some cockpit binaries rely on specific paths in usr/share or libexecdir requiring the paths to be resolved correctly."